tests/app/service/test_rest.py

```diff
@@ -2470,23 +2471,55 @@ def test_is_email_from_unique_returns_200_if_unique(admin_request, notify_db, no
 
 
 @pytest.mark.parametrize('name, email_from',
-                         [("UNIQUE", "unique"),
-                          ("Unique.", "unique"),
+                         [("Unique.", "unique"),
                           ("**uniQUE**", "unique")
                           ])
-def test_is_service_name_unique_returns_200_with_name_capitalized_or_punctuation_added(
+def test_is_service_name_unique_returns_200_with_punctuation_added(
     admin_request,
     notify_db,
     notify_db_session,
     name,
     email_from
 ):
-    create_service(service_name='unique', email_from='unique')
+    service = create_service(service_name='unique', email_from='unique')
```

Comment:
I think you're not using the `email_from` from your parameters here and you should

Reply:
removed unused param